lib/kubernetes-deploy/kubeclient_builder.rb

```diff
@@ -28,7 +29,7 @@ def build_v1beta1_kubeclient(context)
     end
 
     def _build_kubeclient(api_version:, context:, endpoint_path: nil)
-      config = Kubeclient::Config.read(ENV.fetch("KUBECONFIG"))
+      config = GoogleConfig.read(ENV.fetch("KUBECONFIG"))
```

Comment:
I dont have any particular attachment to the name, though I'm not a huge fan of enormous names either.